api/JavascriptBindings.md

```diff
@@ -131,23 +133,55 @@ Currently this function binds only methods of an object. Example:
 	myobject.someMethod();
 ```
 
-Currently when binding object only methods are binded, I decided not to bind properties of the object, as they would be binded by copying value and this might be confusing, as accessing object's property from javascript might give a different value during runtime then the real value when getting the property from python runtime. Only object's methods and functions can be binded by reference. Still you can bind object's properties if you like, you can find useful method IsValueAllowed() to check which properties can be binded, of course doing it this way will not allow you to access properties through "window.myobject.property", you can only bind to the "window" object so you should imitate some kind of namespace, so that accessing property would be through "window.myobject_property" or "myobject_property" as window prefix is always optional. Use dir() function to list object's properties. Example code:
+By default when binding object only methods are bound unless `allow_properties` is explicitly set. This was done as properties can currently only be bound by copying value during the binding process. It is expected this could be confusing as accessing object's property from javascript might give a different value during runtime than the real value when getting the property from python runtime if either end has been changed in the mean time. Only object's methods and functions can be bound by reference currently.  
+If so desired the properties can be bound just be aware that the javascript object will only be able to access the values as they existed at the time binding was performed. Changes to the javascript or python copies of the properties will not be reflected on the other.
 
 ```
-import types
-
-	for name in dir(myobject):
-		if name[0] == '_': # ignore private attributes
-			continue
-		attr = getattr(myobject, name)
-		# Do not bind: functions, methods - this check is necessary as IsValueAllowed is true for these.
-		if type(attr) not in (types.FunctionType, types.MethodType):
-			if bindings.IsValueAllowed(attr):
-				bindings.SetProperty("myobject_"+name, attr)
+# In python:
+    class MyObject(object):
+        def __init__(self):
+            self.definition = "a property"
+
+        @property
+        def py_prop(self):
+            return True
+
+        def someMethod(self):
+            print("someMethod() called")
+
+	myobject = MyObject()
+	bindings.SetObject("myobject", myobject, allow_properties=True)
+	browser.SetJavascriptBindings(bindings)
+	
+	# js will not see this change unless you call browser.Rebind()
```

Comment:
Regarding the value changing during reload/navigation - this seems to not be true, because I've looked at the code and javascript bindings seem to be sent to the Renderer process only when SetJavascriptBindings/Rebind are being called (best would be to test to make sure). However that doesn't make the issue less important, as this still can lead to scenarios when one browser has the js bindings with its original value and the other browser window has a changed value (eg. popup).